_observing-your-data/ad/index.md

```diff
@@ -119,50 +119,41 @@ After you define the detector, the next step is to configure the model.
 
 ## Step 2: Configure the model
 
-1. Add features to your detector.
+Add features to your detector. A _feature_ is any field in your index that you want to analyze for anomalies. A detector can discover anomalies across one or more features. 
 
-A _feature_ is any field in your index that you want to analyze for anomalies. A detector can discover anomalies across one or more features. You must choose an aggregation method for each feature: `average()`, `count()`, `sum()`, `min()`, or `max()`. The aggregation method determines what constitutes an anomaly.
+You must choose an aggregation method for each feature: `average()`, `count()`, `sum()`, `min()`, or `max()`. The aggregation method determines what constitutes an anomaly. For example, if you choose `min()`, the detector focuses on finding anomalies based on the minimum values of your feature. If you choose `average()`, the detector finds anomalies based on the average values of your feature.
```

Comment:
Can you add if users want to use any other method outside the 5 predefined method like average(), they can use a custom expression (a.k.a., JSON aggregation query below)?